tests/PHPStan/Analyser/data/math.php

```diff
@@ -109,4 +109,20 @@ public function doSit(int $i, int $j): void
 		assertType('int', $i - $j);
 	}
 
+	public function unionSum(): void
```

Comment:
When solving a specific issue, please always create a new file called `bug-xxxx.php`, thanks. The namespace in the file should also match.

Reply:
done